framework/tlvf/AutoGenerated/src/tlvf/ieee_1905_1/tlvDeviceInformation.cpp

```diff
@@ -156,19 +180,28 @@ bool tlvDeviceInformation::init()
     }
     if(m_length && !m_parse__){ (*m_length) += sizeof(sMacAddr); }
     if (!m_parse__) { m_mac->struct_init(); }
-    m_info_length = (uint8_t*)m_buff_ptr__;
-    if (!m_parse__) *m_info_length = 0;
+    m_local_interface_list_length = (uint8_t*)m_buff_ptr__;
```

Comment:
C-style cast again ...
Also see my previous remarks regarding the need to even have this variable